services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -84,7 +89,22 @@ export const AddEditReportModal = ({
 
   // MCPAR report payload
   const prepareMcparPayload = (formData: any) => {
-    const programName = formData["programName"];
+    const newOrExistingProgram = formData["newOrExistingProgram"];
+    const existingProgramNameSelection = formData[
+      "existingProgramNameSelection"
+    ] || {
+      label: dropdownDefaultOptionText,
+      value: "",
+    };
+    const existingProgramNameSuggestion =
+      formData["existingProgramNameSuggestion"] || "";
+    const newProgramName = formData["newProgramName"] || "";
```

Comment:
suggestion: could we use `?.` here and let it be undefined?

Reply:
100% the reason why its like that. Didn't want to rock the boat.